product_multi_category/models/product.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2009 Akretion,Guewen Baconnier,Camptocamp,Avanzosc,Sharoon Thomas,Sodexis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    categ_ids = fields.Many2many(
```

Comment:
Why do you remove categ_id = fields.Many2one(string='Pricing/Primary Category') ?

Reply:
Ok, I tested it and seems that it is not necessary